<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_chokolad">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>шоколад</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_caremel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>карамель</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_honey">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>мёд</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_raspberry">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>малина</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_peach">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>персик</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_strawberry">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>клубника</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Дополнения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icecream">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Мороженное:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_photo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>build/Desktop-u041eu0442u043bu0430u0434u043au0430/ice.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчёт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>260</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>build/Desktop-u041eu0442u043bu0430u0434u043au0430/ice.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <zorder>checkBox_chokolad</zorder>
   <zorder>checkBox_caremel</zorder>
   <zorder>checkBox_honey</zorder>
   <zorder>checkBox_raspberry</zorder>
   <zorder>checkBox_peach</zorder>
   <zorder>checkBox_strawberry</zorder>
   <zorder>label_Dop</zorder>
   <zorder>label_icecream</zorder>
   <zorder>pushButton</zorder>
   <zorder>_photo</zorder>
   <zorder>label</zorder>
   <zorder>textBrowser_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
